From ad20049c04c39be7256d8026e6f3b15e636847be Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Thu, 14 Mar 2013 11:31:05 +0200
Subject: [PATCH 28/63] rm: rename `librm' UT to `rm-ut'

`lib' prefix is not applicable. The naming convention for unit tests
is <name>-ut.
---
 rm/ut/rmut.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rm/ut/rmut.c b/rm/ut/rmut.c
index d7f830e..c556fb4 100644
--- a/rm/ut/rmut.c
+++ b/rm/ut/rmut.c
@@ -130,7 +130,7 @@ void rm_utdata_fini(struct rm_ut_data *data, enum obj_type type)
 }
 
 const struct m0_test_suite rm_ut = {
-	.ts_name = "librm",
+	.ts_name = "rm-ut",
 	.ts_tests = {
 		{ "api", rm_api_test },
 		{ "lcredits", local_credits_test },
-- 
1.8.3.2

